USER_GUIDE.md

```diff
@@ -1,7 +1,166 @@
 # User Guide (1.3.0-rc)
 Primitive guide for users 1.3.0 and above (need to expand)
 
-# Yaml configuration deployment
+## Applying the Plugin
+For both _standard_ and _flexible_ applications, include the plugin in your buildscript
+
+```
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+  dependencies {
+    classpath "com.google.cloud.tools:appengine-gradle-plugin:<version>"
+  }
+}
+
+apply plugin: "com.google.cloud.tools.appengine"
+```
+
+The [Cloud SDK](https://cloud.google.com/sdk) is required for this plugin to 
+function. Download and install it before running any tasks.
+
+---
+
+## App Engine Standard
+The plugin will include App Engine standard features if you include an `appengine-web.xml`
```

Comment:
`The plugin will include App Engine standard features` -> `The plugin will treat the project as targeted for App Engine standard environment`?